examples/web/index.html

```diff
@@ -245,6 +246,18 @@ <h3>Stacked blocks</h3>
           </ul>
         </div>
       </section>
+
+      <section>
+        <div class="Container">
+          <h2>Helpers</h2>
+          <h3 class="typography-heading-small-text">Heading Small Text</h3>
```

Comment:
I know, it's work, but… Can we please have here all available styles?

<img width="707" alt="obrazek" src="https://user-images.githubusercontent.com/5614085/156644228-6fdc6b5b-2e4d-4d7a-917e-afdf6ed1d03b.png">